infrastructure/rdfdb/cdk/lib/ecs-stack.js

```diff
@@ -133,7 +133,7 @@ const custom = require('aws-cdk-lib/custom-resources')
     userData.addCommands(scriptWithVolume)
 
     const autoScalingGroup = new autoscaling.AutoScalingGroup(this, 'rdf4jAutoScalingGroup', {
-      instanceType: ec2.InstanceType.of(ec2.InstanceClass.T3, ec2.InstanceSize.MEDIUM),
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.R5, ec2.InstanceSize.LARGE),
```

Comment:
This isn't going to incur us any significant cost increases is it?

Reply:
I'll run this by John, but we can definitely test with smaller machines now that there is an index in place see the differences.

Reply:
I already dropped it down from xlarge to just large.    Also something worth remembering is that we don't scale up, so having something slightly larger might be worthwhile.